src/templates/_common/components/forms.jsx

```diff
@@ -134,6 +131,9 @@ export const SubmitButton = ({
                     {text}
                 </button>
             </div>
+            { !no_error &&
+                <p id={msg_id || 'msg_form'} className={`${is_centered ? 'center-text' : ''} error-msg no-margin invisible`} />
```

Comment:
this will cause the change for all our form errors, are we sure about it?

Reply:
yes, but @ashkanx could you please confirm?